configs/experimental/windows/0.1.0/bazel_experimental/CROSSTOOL

```diff
@@ -242,7 +242,7 @@ cxx_builtin_include_directory: "C:\\Program Files (x86)\\Windows Kits\\8.1\\incl
 # This is a workaround for https://github.com/bazelbuild/bazel/issues/5087.
 cxx_builtin_include_directory: "C:\\tmp\\lre"
 cxx_builtin_include_directory: "C:\\botcode\\w\\ir\\run"
-cxx_builtin_include_directory: "C:\\botcode\\w\\run"
+cxx_builtin_include_directory: "C:\\botcode\\w"
```

Comment:
So you mean that at this point the crosstool is dependent on how we are doing things in the rbe bots? seems slightly problematic to me (fine for experimental though).

Reply:
https://github.com/bazelbuild/bazel/issues/5087 has all the details, but it's an open bug with input validation. cxx_builtin_include_directory disables that validation check for where rbe bots do their work